packages/elements/src/components/ui/action-menu/ActionMenuItem.module.css

```diff
@@ -78,6 +78,11 @@
   }
 
   &.standard {
+    &:disabled:focus-visible {
```

Comment:
Moving `&:disabled:focus-visible` will do nothing for us. This has something to do with making sure the `:focus-visible` styling isn't applied if the button is `:disabled`, so if anything, moving this risks breaking that.

What you need to do is put the `&:disabled, &[aria-disabled="true"]` styling below the `&.danger` etc so that the `:disabled` styling will be read last and override the variant styling.

Right now, disabled items don't look disabled at all:
![image](https://user-images.githubusercontent.com/6987939/213193060-c30d95eb-f25d-4aca-84dd-51c8d31d9881.png)

![image](https://user-images.githubusercontent.com/6987939/213193035-e5b2d11a-5a29-403c-8a00-0e33e23add93.png)